### ISSUE TITLE
cannot click backbutton

### ISSUE THREAD

**Issue by feriferdinan** [georstat/react-native-image-gallery#11]
please allow back button

**Comment by AndreanaPerla** [comment]
back button please

**Comment by likon416** [comment]
yeah, I have the same problem on android, I can't connect the back button to close the gallery. Did anyone solve this somehow?

**Comment by PimiYvan** [comment]
I use renderHeaderComponent props with an icon which help me to close 
` renderHeaderComponent={(_image , currentIndex) => (
      <SafeAreaView style={{flex:1, flexDirection: 'row', justifyContent: 'flex-end'}}>
          
          {/* <Text style={styles.text}>Share</Text> */}
          <IconButton icon={'close'} color='red' onPress={closeGallery}/>
          
      </SafeAreaView>                            
  )`

**Comment by cbjs** [comment]
+1
